app/views/admin/logs/_log.html.slim

```diff
@@ -2,7 +2,7 @@
   .panel-heading
     .row
       .col-sm-10
-        b => log.date
+        b => log.date.try(:strftime, t('time.formats.date_only'))
```

Comment:
I believe the Rails way to do this is to use the `l` helper:

http://guides.rubyonrails.org/i18n.html#adding-date-time-formats

Instead of `try` you will need to use a conditional operator, like `log.date ? l(log.date) : ""`